internal/pkg/service/buffer/event/sender.go

```diff
@@ -32,7 +31,7 @@ type Params struct {
 	Stats      model.Stats
 }
 
-func (s *Sender) SendSliceUploadEvent(ctx context.Context, start time.Time, errPtr *error, slice model.Slice) {
+func (s *Sender) SendSliceUploadEvent(ctx context.Context, api *keboola.API, start time.Time, errPtr *error, slice model.Slice) {
```

Comment:
Proto se teď sem dosazuje při posílání eventu, protože dřív přístup k tokenu není.

Reply:
:+1: vyriesil si to dobre